modules/ingest-common/src/main/java/org/elasticsearch/ingest/common/ScriptProcessor.java

```diff
@@ -80,7 +84,7 @@ public Factory(ScriptService scriptService) {
         @Override
         public ScriptProcessor create(Map<String, Processor.Factory> registry, String processorTag,
                                       Map<String, Object> config) throws Exception {
-            String lang = readStringProperty(TYPE, processorTag, config, "lang");
+            String lang = readStringProperty(TYPE, processorTag, config, "lang", "painless");
```

Comment:
@rjernst simply because I was not aware there was such default behavior in the ScriptService. I will move defaulting responsibility to the ScriptService, will update. thanks!